src/config/emailContent.js

```diff
@@ -0,0 +1,27 @@
+const CONTACT_MESSAGE_FIELDS = {
+  first_name: 'First Name: ',
+  last_name: 'Last Name: ',
+  mobile_phone: 'Phone Number: ',
+  email: 'Email: ',
+  dob: 'Date of Birth: ',
+  requested_date: 'Requested Date: ',
+  requested_time: 'Requested Time: ',
+  appointment_type: 'Appointmet Type: ',
+  description: 'Description: ',
+  is_emergency: 'Is this an emergency? ',
+};
+
+export const generateEmailContent = (data) => {
```

Comment:
Right now, `data` argument has a type of `any`. It will be great to explicitly define its type to `Appointment`.

Reply:
@5hraddha I was struggling a bit with the typescript. I was getting some errors on CONTACT_MESSAGE_FIELDS that were not allowing me to make commits so I kept the file as js for now.

Reply:
Hey @Amanda2900,
Just tried this code where defined the type for the keys and values of the constant `CONTACT_MESSAGE_FIELDS` explicitly so that Typescript should have 'peace' 😅 that its always going to a string as key and value. It seems to solve the problem. Please, try and let me know if that works. 

```ts
type ContactMessageFields = {
  [key: string]: string;
};
const CONTACT_MESSAGE_FIELDS: ContactMessageFields = {
  first_name: 'First Name: ',
  last_name: 'Last Name: ',
  mobile_phone: 'Phone Number: ',
  email: 'Email: ',
  dob: 'Date of Birth: ',
  requested_date: 'Requested Date: ',
  requested_time: 'Requested Time: ',
  appointment_type: 'Appointmet Type: ',
  description: 'Description: ',
  is_emergency: 'Is this an emergency? ',
};
```

Reply:
Thank you for this! I made the changes that you suggested and typescript has stopped yelling at me :)